### DISCUSSION TITLE
gagal menambahkan group

### DISCUSSION THREAD

**Discussion by newfaboc** [kirimwa-id/forum#79] [Tanya Jawab]
hi rio,

saya telah berhasil menambahkan 1 group yang beranggotakan 5 orang.
namun saat saya menambahakn group yang beranggotakan 107 orang tidak muncul id nya.

apakah ada limit anggota kah untuk men syncronize gorup ?

Terima kasih

**Comment by rioastamal** [comment]
Halo mas @newfaboc,

Harusnya tidak ada masalah saya memiliki banyak grup dengan anggota diatas 200+ dan ID-nya selalu muncul. Apakah nomor WA yang digunakan pada grup tersebut tidak diblokir oleh admin dsb. Coba lakukan `/reconnect` dan tunggu hingga device siap kemudia coba call ulang API groups.

Jika masih bermasalah, bisa tolong ditampilkan screenshotnya. (Blur atau blok saja bagian yang sensitif).

**Reply by newfaboc** [reply]
ternyata harus di reconnect. dan berhasil.

terima kasih mas rio
